ci/build.sh

```diff
@@ -1,80 +1,21 @@
 #!/bin/bash
-set -e
+set -eu
 
-# You can run it from any directory.
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+rustup self update
```

Comment:
this installs latest stable version which breaks what I did before — installed specific version of Rust and its tooling

Reply:
Travis installs the specific version for you. 

```
curl -sSf https://build.travis-ci.org/files/rustup-init.sh | sh -s -- --default-toolchain=$TRAVIS_RUST_VERSION -y
```